neuromation/cli/job.py

```diff
@@ -898,7 +898,7 @@ def format_fail(job: str, reason: Exception) -> str:
     return (
         local_nmrc_path,
         Volume(
-            storage_path=storage_nmrc_folder,
+            storage_uri=URL(storage_nmrc_folder),
```

Comment:
I would make the `storage_nmrc_folder` variable of type URL (line 885).

Reply:
Nice idea! Done